bugbug/bugzilla.py

```diff
@@ -344,6 +344,31 @@ def get_product_component_count(months: int = 12) -> dict[str, int]:
     return bugs_number
 
 
+def get_active_product_components(products=[]) -> set[tuple[str, str]]:
+    r = utils.get_session("bugzilla").get(
+        "https://bugzilla.mozilla.org/rest/product",
+        params={
+            "type": "accessible",
+            "include_fields": [
+                "name",
+                "is_active",
+                "components.name",
+                "components.is_active",
+            ],
+            "names": products,
+        },
+        headers={"X-Bugzilla-API-Key": Bugzilla.TOKEN, "User-Agent": "bugbug"},
+    )
+    r.raise_for_status()
```

Comment:
Maybe we could at some point use the `BugzillaProduct` class to fetch components. We do not need to do it now since we already using the API directly in `get_component_team_mapping()`. We could file a follow-up issue to maintain consistency.

Reply:
Makes sense, filed #3551.